### PR TITLE
ex 16 done

### PR DESCRIPTION
**Pull request by AndreaBLS**

### CODE REVIEW

File: 16-PB-conditionals-pt2/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+// 1. Store Mark's and John's mass and height in variables.
+let andrea = { firstName: "Andrea", mass: 72, age: 29, height: 1.75 };
+let willy = { firstName: "Willy", mass: 85, age: 30, height: 1.69 };
+
+// 2. Calculate both their BMIs and store their BMIs in variables. 
+let andreaBMI = BMIcalc(andrea.mass, andrea.height);
+let willyBMI = BMIcalc(willy.mass, willy.height);
+console.log(`Andrea's BMI is ${andreaBMI} and Willy's BMI is ${willyBMI}`);
+
```
Comment (Ungoot) [inline 16-PB-conditionals-pt2/index.js:21]: As you are already using objects you can make these variables part of your objects and call them from inside your objects later.

**Comment by Ungoot** [comment]
Like this;

```
// 2. Calculate both their BMIs and store their BMIs in variables. 
andrea.andreaBMI = BMIcalc(andrea.mass, andrea.height);
willy.willyBMI = BMIcalc(willy.mass, willy.height);
console.log(`${andrea.firstName}'s BMI is ${andrea.andreaBMI} and ${willy.firstName}'s BMI is ${willy.willyBMI}`)

**Comment by Ungoot** [comment]
Good job btw :D

**Comment by AndreaBLS** [comment]
thanks Bleda for the feedback & suggestions :)

**Pull request closed without merging**